prac_08/convert_miles_km.py

```diff
@@ -0,0 +1,36 @@
+"""CP1404 practical 8 - Do-from-scratch 1: Miles to Kilometres Converter"""
+from kivy.app import App
+from kivy.lang import Builder
+from kivy.properties import StringProperty
+
+MILES_TO_KM_FACTOR = 1.609344
+
+
+class MilesConversionApp(App):
+    """Kivy App for converting miles to kilometres."""
+    result = StringProperty()
```

Comment:
Good use of a StringProperty()

Reply:
Thank you! That one definitely took some time to figure out.